File: en/java-spring-demo/src/main/java/com/itangcent/springboot/demo/common/model/TestLogBean.java
package com.itangcent.springboot.demo.common.model;

/**
 * Test Log Bean Class
 * Used to demonstrate the usage of the transient keyword
 * Contains examples of handling sensitive information
 */
public class TestLogBean {
    /**
     * Username
     * Normal serialization and persistence
     */
    private String userName;

    /**
     * Password
     * Marked with transient, excluded from serialization
     * Used to protect sensitive information
     */
    private transient String password;

    public String getUserName() {
        return userName;
    }

    public void setUserName(String userName) {
        this.userName = userName;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }
} 

File: en/kotlin-spring-demo/src/main/kotlin/com/itangcent/common/model/TestLogBean.kt
package com.itangcent.common.model

/**
 * Test Log Bean Class
 * Used to demonstrate @Transient annotation usage
 * Contains examples of handling sensitive information
 */
data class TestLogBean(
    /**
     * Username
     * Normal serialization and persistence
     */
    var userName: String? = null,

    /**
     * Password
     * Marked with @Transient, not included in serialization
     * Used to protect sensitive information
     */
    @Transient
    var password: String? = null
) 